util/reggen/gen_cheader.py

```diff
@@ -89,6 +89,7 @@ def gen_cdefine_register(outstr, reg, comp, width, rnames):
         gen_define(
             defname, ['id'],
             '(' + as_define(comp) + '##id##_BASE_ADDR + ' + hex(offset) + ')'))
+    genout(outstr, gen_define(defname + '_REG_OFFSET', [], hex(offset)))
```

Comment:
Is this an index or an offset?

Reply:
I *believe* this is an offset (which is always word-aligned...)

Reply:
This came up I think because of the behavior of mmio's read/write functions.  These are in fact offsets, given the behavior of mmio now.